extension/fts/src/utils/fts_utils.cpp

```diff
@@ -63,6 +72,10 @@ std::vector<std::string> FTSUtils::stemTerms(std::vector<std::string> terms,
         if (isConjunctive && checker.isStopWord(term)) {
             continue;
         }
+        if (isQuery && hasWildcardPattern(term)) {
```

Comment:
We should still stem the term for now (for example: learns -> learn, studys -> study )